mitmproxy/tools/cmdline.py

```diff
@@ -387,6 +388,11 @@ def proxy_options(parser):
         action="store", type=str, dest="upstream_bind_address",
         help="Address to bind upstream requests to (defaults to none)"
     )
+    group.add_argument(
+        "--keep-host-header",
+        action="store_true", dest="keep_host_header",
+        help="Keep the host header as proxy addres"
```

Comment:
We should be a bit more descriptive here, something along this line:
`Keep the original host header in reverse-mode. The default is to modify the host header to reflect the upstream server address during reverse proxying request.`
@mhils ?